INN/umbrella-sfpublicpress
# Homepage and header spacing updates

## Changes

This pull request makes the following changes:

- Decreases site header margin down to 1.5rem
- Removes padding from the main navbar
- Reduces bottom-margin styling for homepage widget titles to 2rem from 3rem
- Decreases `before-footer` wrapper area bottom padding to 1rem

![screencapture-sfpp-test-2020-05-28-15_46_50](https://user-images.githubusercontent.com/18353636/83186403-785f4880-a0fa-11ea-8f61-1788d45b7acf.png)

## Why



For #76

## Testing/Questions

Features that this PR affects:

- Site header spacing
- Main navbar spacing
- Homepage widget title spacing
- `before-footer` area bottom spacing


Questions that need to be answered before merging:


Steps to test this PR:

1. View the homepage on desktop and mobile and make sure none of the spacing updates messed anything up
